### PR TITLE
Update flaw creation scenario to check alert

### PR DESCRIPTION
**Pull request by RichardXuan**
Fixes OSIDB-2957

### CODE REVIEW

File: features/pages/flaw_detail_page.py [hunk 1]
```diff
@@ -130,14 +134,15 @@ def __init__(self, driver):
...
-        "ManageTrackers":("XPATH", "//button[contains(text(), 'Manage Trackers')]"),
+        "ManageTrackers": ("XPATH", "//button[contains(text(), 'Manage Trackers')]"),
         "affectUpstreamCheckbox": ("XPATH", "(//input[@class='osim-tracker form-check-input'])[1]"),
         "fileSelectedTrackers": ("XPATH", "//button[contains(text(), 'File Selected Trackers')]"),
         "trackersFiledMsg": ("XPATH", "//div[contains(text(), 'trackers filed')]"),
         "disabledfileSelectTrackers": ("XPATH", "//button[contains(text(), 'File Selected Trackers') and @disabled='']"),
         "trackerCount": ("XPATH", "//span[contains(text(), '2 trackers')]"),
         "trackerJiraSummary": ("XPATH", "//summary[contains(text(), 'rhel-8')][1]"),
         "trackerBzSummary": ("XPATH", "//summary[contains(text(), 'rhcertification')][1]"),
+
```
Comment (huiwangredhat) [inline features/pages/flaw_detail_page.py:145]: Would you like to remove it?

File: features/pages/flaw_detail_page.py [hunk 2]
```diff
@@ -106,8 +106,12 @@ def __init__(self, driver):
 
         "bottomBar": ("XPATH", "//div[@class='osim-action-buttons sticky-bottom d-grid gap-2 d-flex justify-content-end']"),
         "bottomFooter": ("XPATH", "//footer[@class='fixed-bottom osim-status-bar']"),
+
```
Comment (huiwangredhat) [inline features/pages/flaw_detail_page.py:109]: same with line 149.

**Review by huiwangredhat — APPROVED** [review]
Others LGTM.

File: features/pages/flaw_detail_page.py (hunk 1 above)
Comment (RichardXuan) [reply features/pages/flaw_detail_page.py:145]: Sure

**Pull request merged**